nucliadb/src/nucliadb/search/search/chat/prompt.py

```diff
@@ -721,7 +722,9 @@ class PromptContextBuilder:
     def __init__(
         self,
         kbid: str,
-        find_results: KnowledgeboxFindResults,
+        main_results: KnowledgeboxFindResults,
+        prequeries_results: Optional[list[PreQueryResult]] = None,
+        main_query_weight: float = 1.0,
```

Comment:
I think this value should be more visible to the user, either with documentation or with a parameter.

Pre-query weights are a percentage, so 1 and 2 is the same as 10 and 20. But the result order will be different for the main query depending on the pre-query weights

Reply:
there is a parameter in fact! I also add param documentation. let me know if its not clear enough.

and yes, the weight of the main query is just like any other prequery

Reply:
Oh I missed it, sorry!